static/src/javascripts-legacy/bootstraps/standard/main.js

```diff
@@ -85,53 +85,6 @@ define([
             }
         });
 
-        /*
```

Comment:
Yeah, this logic now happens in [`inlineJSNonBlocking.scala.html`](https://github.com/guardian/frontend/blob/e6c59a2dcd2e3fc4d0a3304520e8868fdc84d21d/common/app/views/fragments/inlineJSNonBlocking.scala.html#L44-L67)